e2e-tests/native/wdio-ios.conf.ts

```diff
@@ -0,0 +1,25 @@
+import { Capabilities } from '@wdio/types/build/Capabilities'
+import { Testrunner } from '@wdio/types/build/Options'
+
+import { config as defaultConfig } from './wdio.conf'
+
+const iosCapabilities: Capabilities = {
+  platformName: 'iOS',
+  // http://appium.io/docs/en/writing-running-appium/caps/
+  'appium:deviceName': 'iPhone 13 Pro Max',
+  'appium:platformVersion': '15.2',
+  'appium:orientation': 'PORTRAIT',
+  'appium:automationName': 'XCUITest',
+  'appium:language': 'EN',
+  /* how to get app path:
+  XCode: Product -> "show build folder in finder" -> drag item in terminal window
+  Command line (native/ios): xcodebuild -scheme integreat-e2e -workspace Integreat.xcworkspace ONLY_ACTIVE_ARCH=NO -sdk iphonesimulator -configuration Debug -showBuildSettings | grep -m 1 "BUILT_PRODUCTS_DIR" | grep -oEi "\/.*" */
+  'appium:app':
+    '/Users/afischer/Library/Developer/Xcode/DerivedData/Integreat-enomkojtzvcuyvfzikuktexfnnki/Build/Products/Debug-iphonesimulator/Integreat.app',
```

Comment:
Maybe some finds out how to pass this using the command line command. Not sure if the effort is worth to because it's just for local testing